bigquery/snippets/snippet.go

```diff
@@ -379,6 +379,101 @@ func exportToGCS(client *bigquery.Client, datasetID, tableID, gcsURI string) err
 	return nil
 }
 
+func exportSampleTableAsCSV(client *bigquery.Client, gcsURI string) error {
```

Comment:
Can we test these new functions?

Reply:
They're invoked in snippet_test.go.  What kind of test are you looking for?  Probe the bucket for the file(s)?

Reply:
Whoops! You're absolutely right - I was looking at a subset of changes. LGTM.